src/ProxyDecoratedCalls.php

```diff
@@ -4,9 +4,6 @@
 
 namespace Yiisoft\Yii\Debug;
 
-/**
- * @property object $decorated
```

Comment:
Why?

Reply:
This causes many psalm errors in classes used trait.